exercises/minesweeper/description.md

```diff
@@ -1,25 +1,35 @@
-Add the numbers to a minesweeper board.
+Add the mine counts to a completed Minesweeper board.
 
 Minesweeper is a popular game where the user has to find the mines using
 numeric hints that indicate how many mines are directly adjacent
 (horizontally, vertically, diagonally) to a square.
 
 In this exercise you have to create some code that counts the number of
-mines adjacent to a square and transforms boards like this (where `*`
-indicates a mine):
-
-    +-----+
-    | * * |
-    |  *  |
-    |  *  |
-    |     |
-    +-----+
-
-into this:
-
-    +-----+
-    |1*3*1|
-    |13*31|
-    | 2*2 |
-    | 111 |
-    +-----+
+mines adjacent to a given empty square and replaces that square with the
+count.
+
+The board is a rectangle composed of blank space (' ') characters. A mine 
+is represented by an asterisk ('\*') character.
```

Comment:
regarding https://github.com/exercism/problem-specifications/pull/1602#discussion_r332113861 - I'd like to broach the possibility of writing this using backticks. How would that look?

Reply:
We could do backticks around quotes (ie `’ ‘` and `’*’`, but I’m uncertain that’s any form of improvement on the escaped form. It also implies that the quotes are meant to be included in the board strings, leading to the same problem as the border by another angle. The quotes are necessary to make the blank space clear, as ` ` is not going to render very distinctly in all possible readers. Personally I think this fix would cause more significant issues than the one it addresses.

At the end of the day these _are_  markdown files. We need not — and cannot — fight to retain their readability in plaintext as a priority over their readability in a markdown enabled viewer, since that’s their primary means of consumption in the current environment. We _could_ entertain putting a script in the toolchain somewhere that removes all markdown formatting from the generated README that the user downloads (ie README.md -> README.txt), but that would then have to be distinct from the one that’s rendered on the website.

Reply:
:eyes: